conf/resources/data/debt/debt001.json

```diff
@@ -1,38 +1,50 @@
-{
-    "duties": [
-      {
-        "dutyID": "duty01",
-        "originalAmount": 500000,
-        "mainTrans": "1525",
-        "subTrans": "1000",
-        "interestStartDate": "2020-12-20",
-        "dateCreated": "2020-12-20",
-        "paymentHistory": [],
-        "periodEnd": "2021-01-20",
-        "addressPostcode": "BN12 4XH",
-        "postcodeDate": "2019-01-01"
-      },
-      {
-        "dutyID": "duty02",
-        "originalAmount": 400000,
-        "mainTrans": "1525",
-        "subTrans": "1000",
-        "interestStartDate": "2021-02-04",
-        "dateCreated": "2021-02-04",
-        "paymentHistory": [],
-        "periodEnd": "2021-01-20",
-        "addressPostcode": "BN12 4XH",
-        "postcodeDate": "2019-01-01"
-      }
-    ],
-  "breathingSpaces": [
-    {
-      "debtRespiteFrom": "2021-02-04",
-      "debtRespiteTo": "2021-03-04"
-    },
-    {
-      "debtRespiteFrom": "2021-04-04",
-      "debtRespiteTo": "2021-06-04"
-    }
-  ]
+{
+  "debtItems": [
+    {
+      "dutyID": "duty01",
```

Comment:
are we changing this to dutyId?

Reply:
this is based on the responses given to us by pega